public/app/features/alerting/unified/utils/rule-form.ts

```diff
@@ -138,10 +140,34 @@ export function normalizeDefaultAnnotations(annotations: Array<{ key: string; va
   return orderedAnnotations;
 }
 
+export function getNotificationSettingsForDTO(
+  manualRouting: boolean,
```

Comment:
One non-blocking nit: I feel the `manualRouting` parameter shouldn't be handled in this function
The function could only convert one object to another, and the `manualRouting` logic should be handled in the place calling the `getNotificationSettingsForDTO` function

Reply:
I think it's a good option, but I was thinking more about having a method to create this new `notification_settings` 
 field in the DTO depending on the information we have in the form (`manualRouting` and `contactPoints`)